canonical-web-and-design/jaas.ai
# Fix revision URLs

## Done

- Correctly get the latest revision URLs.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open /u/kubeflow-charmers/kubeflow/bundle/77
- Click on "Latest version" in the bundle header.
- You should be take to the latest version of the bundle.

## Details

- Fixes: #187.
